packages/benchmark.rb

```diff
@@ -3,45 +3,15 @@
 class Benchmark < Package
   description 'A microbenchmark support library from Google'
   homepage 'https://github.com/google/benchmark/'
-  version '1.5.2'
+  @_ver = '1.5.5'
+  version @_ver
   license 'Apache-2.0'
   compatibility 'all'
-  source_url 'https://github.com/google/benchmark/archive/v1.5.2.tar.gz'
-  source_sha256 'dccbdab796baa1043f04982147e67bb6e118fe610da2c65f88912d73987e700c'
+  source_url 'https://github.com/google/benchmark.git'
+  git_hashtag 'v' + @_ver
 
-  binary_url ({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/benchmark/1.5.2_armv7l/benchmark-1.5.2-chromeos-armv7l.tar.xz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/benchmark/1.5.2_armv7l/benchmark-1.5.2-chromeos-armv7l.tar.xz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/benchmark/1.5.2_i686/benchmark-1.5.2-chromeos-i686.tar.xz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/benchmark/1.5.2_x86_64/benchmark-1.5.2-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-    aarch64: '9aa663f4068a79d821bc556f6dc60b0e6ee2278505cd747e94a0fd7750258711',
-     armv7l: '9aa663f4068a79d821bc556f6dc60b0e6ee2278505cd747e94a0fd7750258711',
-       i686: 'e8f37ed6c926979e2c13949f3cf8eaeb9e7e9eedbc751d550ead9680e0a87a5c',
-     x86_64: '71f73a3dc296d91b2a5ed685396cbad25ad69e0d231cbed59a5d44391e98aa31',
-  })
-
-  def self.patch
-    @limitsh = <<~EOF
-      --- a/src/benchmark_register.h
-      +++ b/src/benchmark_register.h
-      @@ -1,6 +1,7 @@
-       #ifndef BENCHMARK_REGISTER_H
-       #define BENCHMARK_REGISTER_H
-
-      +#include <limits>
-       #include <vector>
-
-       #include "check.h"
-    EOF
-    IO.write("limitsh.patch", @limitsh)
-    system 'patch -p 1 -i limitsh.patch'
-  end
-
-  def self.prebuild
-    system "git clone git://github.com/google/googletest.git -b release-1.10.0 --depth 1" # Required for build, won't interfere with the gtest package
-  end
+  extra_url 'https://github.com/google/googletest.git' => 'release-1.11.0'
+  extra_url 'https://raw.githubusercontent.com/skycocker/chromebrew/master/install.sh' => '47b36145a617b2a7e2222184cfa92ebb1c13bba4787c1efd7a0a85d54c211632'
```

Comment:
I'm not seeing where these lines are needed.  Am I missing something?  Wouldn't it be easier to include `depends_on 'gtest'` instead?

Reply:
The gtest source code is required at compile time and isn't installed.